OptimizelySDK/OptimizelyUserContext.cs

```diff
@@ -39,22 +38,17 @@ public class OptimizelyUserContext
         private UserAttributes Attributes;
 
         // set of qualified segments
-        public List<string> QualifiedSegments { get; }
+        private List<string> QualifiedSegments;
 
         // Optimizely object to be used.
         private Optimizely Optimizely;
 
         private ForcedDecisionsStore ForcedDecisionsStore { get; set; }
 
-        public OptimizelyUserContext(Optimizely optimizely, string userId,
-            UserAttributes userAttributes, IErrorHandler errorHandler, ILogger logger
-        ) :
-            this(optimizely, userId, userAttributes, null, errorHandler, logger) { }
+        public OptimizelyUserContext(Optimizely optimizely, string userId, UserAttributes userAttributes, IErrorHandler errorHandler, ILogger logger) :
+            this(optimizely, userId, userAttributes, null, null, errorHandler, logger) { }
 
-        public OptimizelyUserContext(Optimizely optimizely, string userId,
-            UserAttributes userAttributes, ForcedDecisionsStore forcedDecisionsStore,
-            IErrorHandler errorHandler, ILogger logger, List<string> qualifiedSegments = null
-        )
```

Comment:
Not sure but we may need to keep this old API as well. Can you double check if it's safe to remove this?